### PR TITLE
forgot and reset password

### PR DESCRIPTION
**Pull request by fossilblue**
For issue 47 forgotten and reset password.

### CODE REVIEW

File: server/email/config.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 Meteor.startup(function() {
```
Comment (davidtzhaozhao) [inline server/email/config.js]: Hi , I thought Mailgun config elements should put in settings-development( ||  settings- *******).json for some secure reasons.

**Comment by davidtzhaozhao** [comment]
also, get a Mailgun group settings, maybe the app need group sending email

File: server/email/config.js (hunk 1 above)
Comment (fossilblue) [inline server/email/config.js]: Yes, all development settings should be added on a different setting file. I believe the configuration for mailgun wasn't changed after adding the Accounts Meteor package.

**Comment by fossilblue** [comment]
Yes for sending the reset password when requesting it.

**Comment by AndyatFocallocal** [comment]
we have a mailgun account. let me k now what settings are needed

**Pull request closed without merging**